src/analysis/souffle/tests/dp_sensitivity_tests/isOperation.facts

```diff
@@ -0,0 +1,5 @@
+["template_owner", "sql.where", "revenue_table_where1", ["revenue_table", nil], nil]
```

Comment:
One question that I have that I don't know is answered by the current tests: does this approach do the right thing if an input is calculated with a value without a sensitivity score, such as a literal?

Reply:
Answered above, we do check for cases like these too.